test/simperium/channel_test.js

```diff
@@ -416,6 +416,37 @@ describe( 'Channel', function() {
 			channel.handleMessage( 'c:' + JSON.stringify( [{error: 405, id: 'thing', ccids: ['abc']}] ) );
 		} );
 
+		it( 'should stop sending duplicate changes after receiving a 409', done => {
+			const change = {o: 'M', id: 'thing', sv: 1, ccid: 'duplicate', v: diff( {}, {key: 'value'} )};
+
+			channel.localQueue.queue( change );
+
+			channel.once( 'send', () => {
+				// we should sent out our change the first time
+				channel.on( 'error', done );
```

Comment:
Note that I used `bucket.on( 'error' ...` in my test because (and I don't know why) I found that `channel.on( 'error'` listeners never fired.

https://github.com/Simperium/node-simperium/pull/73/files#diff-00ce9d6078e473d0387010bc8c77084cR441

Reply:
`bucket` used in e4d5f70